Docs/sphinx_documentation/source/LinearSolvers.rst

```diff
@@ -388,14 +388,22 @@ to the ghost cell center; :cpp:`maxorder = 3` uses the boundary value and the fi
 Curvilinear Coordinates
 =======================
 
-The linear solvers support curvilinear coordinates including 1D
```

Comment:
The nodal solver supports RZ too, at least partially.  It does require the user to rescale the problem.  But the solver itself also has to do some additional correction. (`git grep Src/LinearSolvers/MLMG`)

Reply:
This is what I was trying to say in the next paragraph, but obviously, I was not clear. I'll try again...

Reply:
That's correct.